inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -680,15 +715,25 @@ void MKLDNNNode::initSupportedPrimitiveDescriptors() {
                 PortConfig portConfig;
                 portConfig.inPlace = -1;
                 portConfig.constant = false;
-                portConfig.desc = MemoryDescUtils::applyUndefinedOffset(*getSrcMemDesc(itpd, i));
+                auto desc = getSrcMemDesc(itpd, i);
+                if (desc->getType() == MemoryDescType::Mkldnn) {
```

Comment:
I would say it is better to check `Blocked` bit since the `MemoryDescUtils::cloneWithUndefStridesAndOffset` call is meaningful only for blocked descriptors, all other will be copied disregard their actual type.